### ISSUE TITLE
The focus in a menu of the primary navigation is not shown

### ISSUE THREAD

**Issue by rautenburger** [swiss/styleguide#550]
**(Only when NVDA screen reader is enabled)**

When using the keyboard to navigate the website, the visual focus is lost when entering a primary navigation menu. 

Tab -> Design principles -> Enter -> Arrow down -> Guidelines (accessible focus not shown)
(Image 1)

Image 1:
![no focus](https://user-images.githubusercontent.com/31723894/30902097-59e28af6-a36a-11e7-9fbb-c1c80fa9a93d.png)

Tested on:
https://q-src.github.io/styleguide/en/
Tested with: 
NVDA screen reader

**Comment by topaxi** [comment]
Fixed in #564
